core/src/main/java/com/linecorp/armeria/client/DefaultClientRequestContext.java

```diff
@@ -229,7 +229,16 @@ private DefaultClientRequestContext(
         }
 
         additionalRequestHeaders = options.get(ClientOptions.HEADERS);
-        customizers = copyThreadLocalCustomizers();
+
+        final Consumer<ClientRequestContext> customizer = options.contextCustomizer();
+        final Consumer<ClientRequestContext> threadLocalCustomizer = copyThreadLocalCustomizer();
+        if (customizer == ClientOptions.CONTEXT_CUSTOMIZER.defaultValue()) {
+            this.customizer = threadLocalCustomizer;
+        } else if (threadLocalCustomizer == null) {
+            this.customizer = customizer;
+        } else {
+            this.customizer = customizer.andThen(threadLocalCustomizer);
```

Comment:
Could you update the Javadoc of `Clients.withContextCustomizer()` and `AbstractClientOptionsBuilder.contextCustomizer()` about which customizer will be evaluated first?